docs/insert-version-url.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+"""
+Insert an entry into the version dropdown json file
+
+Usage: add_version_dropdown.py your.version.here
+"""
+
+import json
+import sys
+import os
+
+
+def create_url_entry(version):
+    url = f"https://docs.determined.ai/{version}/"
+
+    entry = {"version": version, "url": url}
+
+    return entry
+
+
+def insert_entry(json_file, entry, entry_position=1):

```

Comment:
Having `entry_position` as an argument that's never used seems like overkill for this tiny script, and the value could bear to be explained.
```suggestion
def insert_entry(json_file, entry):
    # The current dev version comes first and all released versions are in revers chronological
    # order, so the latest release always goes in index 1.
    entry_position = 1
```

Reply:
I don't mind this. It explains the code better anyways.